app/presenters/tree_builder_policy_simulation.rb

```diff
@@ -1,13 +1,14 @@
+# rubocop: disable ClassLength
 class TreeBuilderPolicySimulation < TreeBuilder
   # exp_build_string method needed
   include ApplicationController::ExpressionHtml
 
   has_kids_for Hash, [:x_get_tree_hash_kids]
 
-  def initialize(name, type, sandbox, build = true, root = nil, root_name, options)
-    @data = root
-    @root_name = root_name
-    @policy_options = options
+  def initialize(name, type, sandbox, build = true, **params)
```

Comment:
Forget it. The magic `**` makes `params` to have a default value of `{}` so my comment does not apply here.